### PR TITLE
Add Hashnode CMS example

### PR DESCRIPTION
**Pull request by Harsh062**
Adding a [blog-starter](https://github.com/vercel/next.js/tree/canary/examples/blog-starter) plus TypeScript boilerplate template wired with [Hashnode](https://hashnode.com) APIs.

### CODE REVIEW

File: examples/cms-hashnode/components/integrations.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    matomoURL,
+    matomoSiteID,
+    fathomSiteID,
+    fathomCustomDomain,
+    fathomCustomDomainEnabled,
+    plausibleAnalyticsEnabled,
+  } = publication.integrations ?? {}
+  const domainURL = new URL(publication.url).hostname
+
+  let fbPixel = `
```
Comment (leerob) [inline examples/cms-hashnode/components/integrations.tsx:19]: Could we strip this example back down to the most minimal example following `cms-starter` + Hashnode?

Comment (Harsh062) [reply examples/cms-hashnode/components/integrations.tsx]: @leerob I have now removed the third party integrations and analytics tracking

Comment (Harsh062) [reply examples/cms-hashnode/components/integrations.tsx]: @leerob Could someone from the team please check this?

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 91856a07e7ff56f55195f55d5bdebc9a08633bf2

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: examples/cms-hashnode/components/integrations.tsx (hunk 1 above)
Comment (Harsh062) [reply examples/cms-hashnode/components/integrations.tsx]: @leerob Could you please provide more insight into why some jobs are failing in this build? The current build results lack sufficient detail for us to diagnose the issue. Locally, both `pnpm build` and `pnpm lint`
Screenshot 2024-06-26 at 12 48 36 PM
Screenshot 2024-06-26 at 12 49 09 PM
 commands run successfully without any errors (Screenshot attached for reference).

**Comment by leerob** [comment]
Thank you for the suggestion. This example is still pretty detailed, which isn't bad in general (nice work!), but not something I want to merge here. I would suggest making a new repo and pushing that example there for the community to see 😄 Could even share back in https://vercel.community/ 🚀

**Pull request closed without merging**